_posts/en/newsletters/2021-11-03-newsletter.md

```diff
@@ -0,0 +1,180 @@
+
+  In addition to direct counterpoints, several advantages of the current
+  system were noted:
+
+    - **Miner anonymity:** with 50,000 or more nodes currently all
+      relaying transactions, it's easy for miners to receive all the
+      information they need by quietly operating one of those nodes.
+      Pseudonymous developer ZmnSCPxj [suggested][zmnscpxj relay] that
+      forcing miners to maintain a persistent identity, even one on an
+      anonymity network like Tor, would make it easier to
+      identify miners and coerce them into censoring some transactions.
+
+    - **Censorship resistance:** anyone today can start a node using
+      basically any computer to receive relayed transactions, connect
+      mining equipment, and begin mining.  Pieter Wuille [notes][wuille
+      relay] that wouldn't be the case with direct miner submission.
+      New miners would need to advertise their nodes in order to receive
+      transactions, but there's no highly accessible way to do that
+      which is resistant to both censorship and [sybil attacks][].  In
+      particular, Wuille notes "ideas for a mechanism for miners to
+      publish their [submission URL] on chain don't help; that [would
+      be] dependent on other miners to not censor the publishing."
+
+    - **Centralization resistance:** Wuille also notes that "the cost and
+      complexity of an additional submission is independent of [each
+      miner's] hashrate, but the benefit is directly proportional [to
+      hashrate".  That would make it "far easier for most wallets to
+      just submit [transactions] to the largest few pools", encouraging
+      centralization.
+
+- **Taproot test vectors:** Pieter Wuille [posted][wuille test] to the
+  Bitcoin-Dev mailing list a set of [test vectors][bips #1225] he's
+  proposing to add the [BIP341][] specification of [taproot][topic
+  taproot].  They focus on "wallet implementations, covering Merkle root
+  / tweak / scriptPubKey computation from key/scripts,
+  sigmsg/sighash/signature computation for key path spending, and
+  control block computation for script path spending."
+
+    The test vectors should be especially useful to developers working
+    on implementations which want to be ready to use taproot [shortly
+    after activation][p4tr waiting].
+
+## Preparing for taproot #20: what happens at activation?
+
+*A weekly [series][series preparing for taproot] about how developers
+and service providers can prepare for the upcoming activation of taproot
+at block height {{site.trb}}.*
+
+{% include specials/taproot/en/20-activation.md %}
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [Bitcoin Core 0.20.2][] is a maintenance release for a previous branch
+  of Bitcoin Core [containing][bcc0.20.2 rn] 0.20.2 minor features and
```

Comment:
I think there's an errant 0.20.2 in here:

```suggestion
  of Bitcoin Core [containing][bcc0.20.2 rn] minor features and
```

Reply:
Good catch. Resolved.